src/Notifications/Notification/AdjustmentCreatedNotification.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Paddle\SDK\Notifications\Notification;
+
+use Paddle\SDK\Entities\Event\EventTypeName;
+use Paddle\SDK\Notifications\Entities\Adjustment;
+use Paddle\SDK\Notifications\Entities\Entity;
+use Paddle\SDK\Notifications\Events\AdjustmentCreated;
+
+final class AdjustmentCreatedNotification extends AdjustmentCreated implements NotificationInterface
+{
+    private function __construct(
+        string $eventId,
+        EventTypeName $eventType,
+        \DateTimeInterface $occurredAt,
+        Adjustment $adjustment,
+        public readonly string $notification_id,
+    ) {
+        parent::__construct($eventId, $eventType, $occurredAt, $adjustment);
+    }
+
+    public function getNotificationId(): string
```

Comment:
We cannot have `public readonly string $notification_id` on the interface, so have added a getter.